test-integration/samples/gradle-jwt-angular.jdl

```diff
@@ -6,7 +6,7 @@ application {
     blueprints [generator-jhipster-micronaut]
     buildTool gradle
     cacheProvider ehcache
-    clientFramework react
+    clientFramework angularX
```

Comment:
I did put a "fix" in place for this sample (that I was going to contribute upstream and forgot... I do that this weekend).  Is this what is making the tests flaky?  https://github.com/jhipster/generator-jhipster-micronaut/tree/master/test-integration/samples/fixes/gradle-jwt-react/src/test/javascript/e2e/modules/account

I'm worried about not having a React client in the CI

Reply:
True, but we right now our react test are very flaky. I hope they will be fixed with https://github.com/jhipster/generator-jhipster/pull/11923.